EMP Torpedo Storage Bays (#5241)
endless-sky/endless-sky: data/remnant/remnant outfits.txt

	"emp torpedo capacity" -1
	description "This torpedo can emit electromagnetic pulses, interfering with a ship's generators."

outfit "EMP Torpedo Storage Bay"
	category "Ammunition"
	licenses
		Remnant
	cost 98000
	thumbnail "outfit/emp rack"
	"mass" 5
	"outfit space" -12
	"energy capacity" 400
	"emp torpedo capacity" 7

> 	description "The EMP Torpedo Storage Bay is used to store extra ammunition for EMP Bays."

Reviewer: ```suggestion
	description "The Storage Bay is used to store extra EMP Torpedoes."
```
Given that the torpedoes are used by the Ka'het as well, I think this fits better

> 	description "The EMP Torpedo Storage Bay is used to store extra ammunition for weapons that use EMP Torpedoes."

Reviewer 2: ```suggestion
	description "The EMP torpedo storage bay is used to store extra ammunition for weapons that use EMP torpedoes."
```

> 	description "In addition to its ammunition capacity, the Storage Bay can also contain a small amount of charge, enough to let an otherwise battery-less ship fire EMP Torpedoes."

Reviewer 2: Smh.
```suggestion
	description "	In addition to its ammunition capacity, the storage bay can also contain a small amount of charge, enough to let an otherwise battery-less ship fire EMP torpedoes."
```
